### PR TITLE
setup-off-on branch is ready for review

### PR DESCRIPTION
**Pull request by hawk198723**
Setup-off-on branch is ready for review. I also added two buttons, will make them functional in the third branch. Please leave a comment and approve this pull request, thank you in advance.  Jason

### CODE REVIEW

File: src/components/SquareBox.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      this.state = {
+        color: ["white", "yellow"],  // array of two colors
+        lightStatus: ["OFF", "ON"],  // array of two status
+        index: 0 // index is the position of the array, in this case: 0 ===> "OFF" ===> "white" (by default)
+      }
+    }
+    // status setter
+    changeStatus = () => {
+        //this.state.index will always change between 0 and 1, since index modular by 2, the result will always be 0 or 1, aka "OFF" and "ON"
+        this.setState({index: (this.state.index + 1) % 2 })
```
Comment (kirkkwang) [inline src/components/SquareBox.js:15]: that's pretty clever, i like this solution!

File: src/components/SquareBox.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        //alert("Yo!")
+    }
+    //create a button, not functional yet
+    addBox = () => {
```
Comment (kirkkwang) [inline src/components/SquareBox.js:19]: Nice, setting up the next branches now

File: src/components/SquareBox.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        // onClick = {this.changeStatus} when the box get clicled, call the changeStatus function
+        // {this.state.lightStatus[this.state.index]} show the text status in the box
+        <>
+         <button onClick = {this.addBox} class = "button">
+             Add Light Box
+         </button>
+         <button onClick = {this.removeBox} class = "button">
+             Remove Light Box
+         </button>
+         <div id = "square"  onClick = {this.changeStatus} style={{background: this.state.color[this.state.index]}}>
```
Comment (kirkkwang) [inline src/components/SquareBox.js:38]: Looks good man, easy to read, good comments throughout the code

**Review by kirkkwang — APPROVED** [review]

File: src/components/SquareBox.js (hunk 1 above)
Comment (hawk198723) [reply src/components/SquareBox.js:15]: Thanks bro, I am so happy it works.

File: src/components/SquareBox.js (hunk 3 above)
Comment (hawk198723) [reply src/components/SquareBox.js:38]: Much appreciated, my brother!

**Pull request merged**